retirement/views.py

```diff
@@ -55,7 +55,10 @@ class RetirementViewSet(viewsets.ModelViewSet):
     serializer_class = serializers.RetirementSerializer
     queryset = Retirement.objects.all()
     permission_classes = (permissions.IsAdminOrReadOnly, )
-    filter_fields = '__all__'
+    filter_fields = {
+        'start_time': ['exact', 'gte', 'lte'],
+        'end_time': ['exact', 'gte', 'lte'],
+    }
```

Comment:
This removes all other possible filters. With this method, we have to explicitly add all fields to enable filtering on them.

Reply:
Yes i know we have the same problem with `TimeslotView` but since i just filter by `end_date` i though it will be better to not add a lot of choice without test on it.

Reply:
With these two lines i'm able to finish the first iteration of development. After that we can take some times to add new filters if needed or take some times to resolve this limitation.

Reply:
I do not agree with that. Why not add all fields now? Have a look at the UserViewSet which does that.

Reply:
If you want to try to add all the fields, with all the tests and finish all the customer request and bugs that we have for the delivery date i think you will need to invest in some good coffee :laughing:

Reply:
You convinced me! :+1: